Misc/NEWS.d/next/Tests/2024-07-22-01-32-00.gh-issue-122102.6sBzoH.rst

```diff
@@ -0,0 +1,2 @@
+Improve the tests for :func:`inspect.ismethoddescriptor` and
```

Comment:
Should I remove this file?

Reply:
(I don't know whether the thumb up by Serhiy was to tell me that the first sentence was correct or if it was for the second sentence...).

@serhiy-storchaka Should we keep it or remove it? (I'm in favor of removing it because I don't think people are very interested in that and it's only about coverage).

Reply:
OK, I remove it then.

Reply:
@serhiy-storchaka Then probably the `skip news` label will be needed, won't it?